Whiskey/Functions/Uninstall-WhiskeyPowerShellModule.ps1

```diff
@@ -39,7 +39,7 @@ function Uninstall-WhiskeyPowerShellModule
 
     Get-Module -Name $Name | Remove-Module -Force
 
-    $modulesRoot = Join-Path -Path $BuildRoot -ChildPath $powerShellModulesDirectoryName
+    $modulesRoot = Join-Path -Path $BuildRoot -ChildPath $psModulesDirectoryName

```

Comment:
Add a `$script:` prefix to indicate this is a global variable for the entire script module?

Reply:
👍